src/nvim/mbyte.c

```diff
@@ -991,40 +991,41 @@ int utf_char2len(const int c)
 /// @param c character to convert to \p buf
 /// @param[out] buf UTF-8 string generated from \p c, does not add \0
 /// @return Number of bytes (1-6).
-int utf_char2bytes(const int c, char *const buf)
+int utf_char2bytes(const int c, char *const buf_in)
 {
+  unsigned *buf = (unsigned *)buf_in;
```

Comment:
This is not going to work.

Reply:
Eh, you're right. Thanks for the heads up.